### PR TITLE
Add redirects for old va.gov pages that we are replacing

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

In the URL mapping spreadsheet, there are current va.gov pages that we want to redirect to current vets.gov pages. This creates client-side redirects for each of those pages.

NOTE: This is not the ideal way of doing these redirects. I think this should be treated as a fallback and that we should try to create real 301 redirects in nginx as our primary solution.

This will also require an nginx config change anyway, to serve `.asp` urls as html.

The source list for these redirects was generated by this code: https://github.com/department-of-veterans-affairs/vets.gov-utility/pull/23/files.
## Testing done

Verified a few pages locally.

## Acceptance criteria
- [x] Pages listed as old va.gov pages redirect to the vets.gov page when opened

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/options.js [hunk 1]
```diff
@@ -76,7 +77,8 @@ function applyEnvironmentOverrides(options) {
 function applyBrandConsolidationOverrides(options) {
   Object.assign(options, {
     contentRoot: '../va-gov',
-    collections: require('./collections/brand-consolidation.json')
+    collections: require('./collections/brand-consolidation.json'),
+    redirects: require('../config/vagovRedirects.json')
```
Comment (ncksllvn) [inline script/options.js]: Since the redirects are set up in Metalsmith, can we store that JSON file in the `script` directory?

**Review by ncksllvn — APPROVED** [review]
Minor suggestion, but LGTM!

File: script/create-redirects.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ function getRedirectPage(redirectToPath) {
   return `
     <!doctype html>
     <head>
-      <meta http-equiv=refresh content="1; url=${absolutePath}">
+      <meta http-equiv=refresh content="0; url=${absolutePath}">
```
Comment (ncksllvn) [inline script/create-redirects.js]: I had that `1` there only to reveal momentarily that we had a link that should be updated, although that isn't an option for sites outside of ours.

File: script/options.js (hunk 1 above)
Comment (jbalboni) [reply script/options.js]: Sure

File: script/create-redirects.js (hunk 2 above)
Comment (jbalboni) [reply script/create-redirects.js]: I can revert that back. From what I read, shorter intervals there made it more likely that you wouldn't lose some of the link ranking in search engines. But given that this is a fallback, maybe that's not necessary.

**Comment by jbalboni** [comment]
I've updated this PR to only cover the www.va.gov domain redirects. Redirects for other domains have to be handled in JS injected into the teamsite pages.

**Pull request merged**